addon/components/utils/templated-input-group.hbs

```diff
@@ -0,0 +1,37 @@
+<div class="fx-col fx-gap-px-6" ...attributes>
+  <div class="fx-col fx-gap-px-3">
+    <div class="fx-row fx-malign-space-between">
+      <div class="font-color-gray-500 font-size-md">
+        <span>{{@title}}</span>{{#if @required}}<span class="font-color-error-500">*</span>{{/if}}
+      </div>
+      <OSS::Link
+        @icon="fa-plus"
+        @label={{t "upf_utils.templated_input_group.insert_variable"}}
+        {{on "click" this.openTemplateVariables}}
+        data-control-name="templated-input-group-insert-variable-link"
+      />
+      <div
+        class="upf-floating-menu upf-floating-menu--{{if this.displayTemplateVariables 'visible' 'hidden'}}"
+        {{on-click-outside this.closeTemplateVariables useCapture=true}}
+        {{attach-element to=(concat "#" @id) placement="bottom-end" offset=0 width="300"}}
+      >
+        {{#each @variables as |variable|}}
+          <div class="upf-floating-menu__item fx-row" role="button" {{on "click" (fn this.insertVariable variable)}}>
+            <span class="text-ellipsis">{{variable}}</span>
+          </div>
+        {{/each}}
+      </div>
+    </div>
+    <span class="font-color-gray-500 text-size-4">{{@subtitle}}</span>
+  </div>
+  <OSS::InputContainer
+    id={{@id}}
+    @value={{this.inputValue}}
+    @placeholder={{@placeholder}}
+    @required={{@required}}
```

Comment:
the OSS::InputContainer doesn't support any `required` argument ^^ so you can remove this line :)